packages/cozy-mjml/src/components/MJDefaults.js

```diff
@@ -22,19 +22,19 @@ class MJDefaults extends core.HeadComponent {
   handler() {
     const { add } = this.context
 
-    const highlight = { color: '#297ef2', 'font-weight': 'bold' }
+    const highlight = { color: '#0a84ff', 'font-weight': 'bold' }
     const primaryLink = {
-      color: '#297ef2',
+      color: '#0a84ff',
       'text-decoration': 'none',
       'font-weight': 'bold'
     }
 
     // Load the Lato font from Google APIs
-    add('fonts', 'Lato', 'https://fonts.googleapis.com/css?family=Lato')
```

Comment:
Looks like the font is named `Inter` and not `inter` (i.e. needs a capital I). The URL returns a 400 response in any case.